### ISSUE TITLE
Loss doesnt goes down

### ISSUE THREAD

**Issue by FacundoMartinezCampos** [ylacombe/finetune-hf-vits#32]
I am finetuning a catalan checkpoint from MMS, with a single speaker dataset of 5 hours, the loss doesnt goes bellow 22, all the characters of my the transcription from my datset exist inside the vocab of the model.

I dont know if i just need more training, how many epochs for 2500 audios is a good amount to get better results than the checkpoint?

I already did 400

**Comment by omerarshad** [comment]
were you able to solve this? I am also finetuning the model on 5k audios, and loss is stuck at 32

**Comment by muhammadsaadgondal** [comment]
Did you guys make it lesser? I got the same problem
My generated audio sounds a little distorted. Is this also because of it. I trained on 100 epochs and 2e-4 lr

**Comment by omerarshad** [comment]
Reducing the weight co-efficients works, but you need to train for a lot more epochs then mentioned, and depends on quality of your dataset.

**Comment by muhammadsaadgondal** [comment]
@omerarshad  I'm training on 40 mins of good quality dataset how many epochs do I need to get clear audio.
Can you give the updated weights? I cant locate the saved weights in my local files.

**Comment by andergisomon** [comment]
> @omerarshad  I'm training on 40 mins of good quality dataset how many epochs do I need to get clear audio.
> Can you give the updated weights? I cant locate the saved weights in my local files.

How long did it take you and what was your hardware setup? I'm wondering if this is possible on a midrange gaming laptop.

**Comment by omerarshad** [comment]
@andergisomon I finetuned it on google colab pro, A100 with batch size of 64 for around 400+ epochs

**Comment by FacundoMartinezCampos** [comment]
lowering the mel weight from 35 to 10 after training 2000 epochs on 1 hour of audio made the step loss go from 16 to 7, i need to finish training to see the results but looks promising

**Comment by FacundoMartinezCampos** [comment]
nope, even when the lossrate dropped from 16 to 7 the quality is still bad, very robotic

**Comment by FacundoMartinezCampos** [comment]
> Reducing the weight co-efficients works, but you need to train for a lot more epochs then mentioned, and depends on quality of your dataset.

which weights?
